python/graphstorm/model/node_gnn.py

```diff
@@ -125,7 +125,7 @@ def forward(self, blocks, node_feats, _, labels, input_nodes=None):
         # weighted addition to the total loss
         return pred_loss + alpha_l2norm * reg_loss
 
-    def predict(self, blocks, node_feats, _, input_nodes):
+    def predict(self, blocks, node_feats, _, input_nodes, return_proba):
```

Comment:
Is this function inherited from one of the mixins (GSgnnModel, GSgnnNodeModelInterface)? If yes we should ensure the arguments are consistent and override at the top level and ensure all concrete implementations are consistent as well.

Reply:
Done